2024-05-02 15:06:07.218 [info] 




2024-05-02 15:06:07.319 [info] Extension host agent started.
2024-05-02 15:06:07.644 [info] [<unknown>][8840ddec][ExtensionHostConnection] New connection established.
2024-05-02 15:06:07.662 [info] [<unknown>][a731083c][ManagementConnection] New connection established.
2024-05-02 15:06:08.727 [info] [<unknown>][8840ddec][ExtensionHostConnection] <17709> Launched Extension Host Process.
2024-05-02 15:06:41.023 [info] [<unknown>][5080d589][ExtensionHostConnection] New connection established.
2024-05-02 15:06:41.042 [info] [<unknown>][5080d589][ExtensionHostConnection] <18200> Launched Extension Host Process.
2024-05-02 15:06:41.046 [info] [<unknown>][80497a2f][ManagementConnection] New connection established.
2024-05-02 15:06:43.658 [error] CodeExpectedError: Could not find pty on pty host
    at D.U (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-05-02 15:11:07.326 [info] New EH opened, aborting shutdown
2024-05-02 15:11:07.331 [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/server/node/server.main.js:191:6058)
    at process.emit (node:events:517:28)
2024-05-02 15:11:41.507 [info] [<unknown>][8840ddec][ExtensionHostConnection] <17709> Extension Host Process exited with code: 0, signal: null.
2024-05-02 15:28:12.233 [info] [<unknown>][70ca831f][ManagementConnection] New connection established.
2024-05-02 15:28:12.243 [info] [<unknown>][b40eb831][ExtensionHostConnection] New connection established.
2024-05-02 15:28:12.271 [info] [<unknown>][b40eb831][ExtensionHostConnection] <906> Launched Extension Host Process.
2024-05-02 15:28:14.356 [error] CodeExpectedError: Could not find pty on pty host
    at D.U (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-05-02 15:28:21.275 [info] [<unknown>][c5b66bb3][ManagementConnection] New connection established.
2024-05-02 15:28:21.282 [info] [<unknown>][479233b7][ExtensionHostConnection] New connection established.
2024-05-02 15:28:21.306 [info] [<unknown>][479233b7][ExtensionHostConnection] <1222> Launched Extension Host Process.
2024-05-02 15:28:23.708 [error] CodeExpectedError: Could not find pty on pty host
    at D.U (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-05-02 15:28:24.837 [error] CodeExpectedError: Could not find pty on pty host
    at D.U (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-05-02 15:28:24.839 [error] CodeExpectedError: Could not find pty on pty host
    at D.U (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateIcon (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:27683)
    at j.<computed> (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-05-02 15:33:12.611 [info] [<unknown>][5080d589][ExtensionHostConnection] <18200> Extension Host Process exited with code: 0, signal: null.
2024-05-02 15:33:21.684 [info] [<unknown>][b40eb831][ExtensionHostConnection] <906> Extension Host Process exited with code: 0, signal: null.
2024-05-02 15:36:16.002 [info] [<unknown>][8d26a6a6][ManagementConnection] New connection established.
2024-05-02 15:36:16.011 [info] [<unknown>][25ccc5e3][ExtensionHostConnection] New connection established.
2024-05-02 15:36:16.038 [info] [<unknown>][25ccc5e3][ExtensionHostConnection] <7755> Launched Extension Host Process.
2024-05-02 15:36:18.104 [error] CodeExpectedError: Could not find pty on pty host
    at D.U (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-05-02 15:36:25.871 [info] [<unknown>][cb09b2a7][ManagementConnection] New connection established.
2024-05-02 15:36:25.879 [info] [<unknown>][88cb8d6e][ExtensionHostConnection] New connection established.
2024-05-02 15:36:25.903 [info] [<unknown>][88cb8d6e][ExtensionHostConnection] <8007> Launched Extension Host Process.
2024-05-02 15:36:28.289 [error] CodeExpectedError: Could not find pty on pty host
    at D.U (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:32471)
    at D.updateTitle (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:27632)
    at j.<computed> (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:47:22875)
    at Object.call (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:36:5137)
    at l.s (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:5145)
    at l.q (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4647)
    at n.value (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:34:4045)
    at t.y (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:665)
    at t.fire (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:29:882)
    at process.ue (/home/mi/.vscode-server/cli/servers/Stable-e170252f762678dec6ca2cc69aba1570769a5d39/server/out/vs/platform/terminal/node/ptyHostMain.js:25:10030)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-05-02 15:41:16.253 [info] [<unknown>][479233b7][ExtensionHostConnection] <1222> Extension Host Process exited with code: 0, signal: null.
2024-05-02 15:41:26.164 [info] [<unknown>][25ccc5e3][ExtensionHostConnection] <7755> Extension Host Process exited with code: 0, signal: null.
2024-05-02 15:48:41.177 [info] [<unknown>][f6ed500c][ManagementConnection] New connection established.
2024-05-02 15:48:41.184 [info] [<unknown>][2ef17727][ExtensionHostConnection] New connection established.
2024-05-02 15:48:41.208 [info] [<unknown>][2ef17727][ExtensionHostConnection] <17159> Launched Extension Host Process.
2024-05-02 15:51:58.390 [info] Getting Manifest... ms-python.python
2024-05-02 15:51:58.515 [info] Installing extension: ms-python.python
2024-05-02 15:52:00.144 [info] Getting Manifest... ms-python.vscode-pylance
2024-05-02 15:52:00.205 [info] Getting Manifest... ms-python.debugpy
2024-05-02 15:52:00.459 [info] Installing extension: ms-python.vscode-pylance
2024-05-02 15:52:00.462 [info] Installing extension: ms-python.debugpy
2024-05-02 15:53:07.419 [info] Extension signature is verified: ms-python.debugpy
2024-05-02 15:53:07.958 [info] Extension signature is verified: ms-python.python
2024-05-02 15:53:08.171 [info] Extension signature is verified: ms-python.vscode-pylance
2024-05-02 15:53:15.315 [info] Extracted extension to file:///home/mi/.vscode-server/extensions/ms-python.debugpy-2024.4.0-linux-arm64: ms-python.debugpy
2024-05-02 15:53:15.349 [info] Renamed to /home/mi/.vscode-server/extensions/ms-python.debugpy-2024.4.0-linux-arm64
2024-05-02 15:53:15.379 [info] Extracting extension completed. ms-python.debugpy
2024-05-02 15:53:30.535 [info] Extracted extension to file:///home/mi/.vscode-server/extensions/ms-python.python-2024.4.1: ms-python.python
2024-05-02 15:53:30.577 [info] Renamed to /home/mi/.vscode-server/extensions/ms-python.python-2024.4.1
2024-05-02 15:53:30.611 [info] Extracting extension completed. ms-python.python
2024-05-02 15:53:41.436 [info] [<unknown>][88cb8d6e][ExtensionHostConnection] <8007> Extension Host Process exited with code: 0, signal: null.
2024-05-02 15:53:45.087 [info] Extracted extension to file:///home/mi/.vscode-server/extensions/ms-python.vscode-pylance-2024.4.1: ms-python.vscode-pylance
2024-05-02 15:53:45.122 [info] Renamed to /home/mi/.vscode-server/extensions/ms-python.vscode-pylance-2024.4.1
2024-05-02 15:53:45.140 [info] Extracting extension completed. ms-python.vscode-pylance
2024-05-02 15:53:45.173 [info] Extension installed successfully: ms-python.debugpy
2024-05-02 15:53:45.175 [info] Extension installed successfully: ms-python.python
2024-05-02 15:53:45.175 [info] Extension installed successfully: ms-python.vscode-pylance
2024-05-02 15:54:09.872 [error] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /usr/lib/python3/dist-packages)
2024-05-02 16:03:27.263 [info] [<unknown>][db7108d1][ManagementConnection] New connection established.
2024-05-02 16:03:27.274 [info] [<unknown>][46256a19][ExtensionHostConnection] New connection established.
2024-05-02 16:03:27.299 [info] [<unknown>][46256a19][ExtensionHostConnection] <27368> Launched Extension Host Process.
2024-05-02 16:03:34.142 [error] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /home/mi)
2024-05-02 16:08:27.660 [info] [<unknown>][2ef17727][ExtensionHostConnection] <17159> Extension Host Process exited with code: 0, signal: null.
2024-05-02 17:32:30.130 [error] [File Watcher (node.js)] Failed to watch /home/mi/.git for changes using fs.watch() (Error: ENOSPC: System limit for number of file watchers reached, watch '/home/mi/.git')
